docs/src/use/configure/configuration-files.md

```diff
@@ -511,14 +511,14 @@ You need to enable this feature through the `unstable_ts_config` feature flag:
 npx eslint --flag unstable_ts_config
 ```
 
-For Deno and Bun, TypeScript configuration files are natively supported; for Node.js, you must install the optional dev dependency [`jiti`](https://github.com/unjs/jiti) in your project (this dependency is not automatically installed by ESLint):
+For Deno and Bun, TypeScript configuration files are natively supported; for Node.js, you must install the following optional dev dependencies in your project (these dependencies are not automatically installed by ESLint):
```

Comment:
npm automatically installs peer dependencies. If we're adding these packages as peer dependencies, then I don't think we need to ask people to install them here?

Reply:
The suggestion in this PR is to add them as optionals:

```json
  "peerDependenciesMeta": {
    "jiti": {
      "optional": true
    },
    "@types/estree": {
      "optional": true
    },
    "@eslint/core": {
      "optional": true
    },
    "@types/json-schema": {
      "optional": true
    }
  },
```

Reply:
So that means npm wouldn't install them automatically?

Reply:
And then wouldn't we also need to install these on Deno and Bun?

Reply:
@nzakas Exactly, only way to make truly optional dependencies, unlike `optionalDependencies`

From [npm docs](https://docs.npmjs.com/cli/v10/configuring-npm/package-json#peerdependenciesmeta):

> allows peer dependencies to be marked as optional. Npm will not automatically install optional peer dependencies

Whereas `optionalDependencies` are only through `npm install --omit=optional` or when:

> you would like npm to proceed if it cannot be found or fails to install